src/06-loops.js

```diff
@@ -83,7 +107,13 @@ function containsWhileLoop(array, target) {}
  *  containsForLoop([ "left", "up", "right" ], "down");
  *  //> false
  */
-function containsForLoop(array, target) {}
+  function containsForLoop(array, target) {
+    for(let n = 0; n <= array.length; n++)
```

Comment:
This for loop is missing curly braces. The code happens works since there is only a single statement inside the for loop, but you should always use curly braces to denote the start and end of the body of a loop.